python/setup.py.in

```diff
@@ -209,8 +209,9 @@ if '${WITH_MKLDNN}' == 'ON':
         command = "patchelf --set-rpath '$ORIGIN/' ${MKLDNN_SHARED_LIB}"
         if os.system(command) != 0:
             raise Exception("patch libmkldnn.so failed, command: %s" % command)
-    package_data['paddle.libs']+=['libmkldnn.so.0' if os.name != 'nt' else ('mkldnn' + ext_name)]
+    package_data['paddle.libs']+=['libmkldnn.so.0','libmkldnn.so.1' if os.name != 'nt' else ('mkldnn' + ext_name)]
```

Comment:
@luotao1 I've got a question about python API. Should I also keep libmkldnn.so.0 here or it's just a problem with CAPI?

Reply:
It's just a problem with CAPI, but it's better to keep the same library between python API and CAPI. Otherwise, it may be confused for developers later. How do you think about it?

Reply:
I really don't like the idea of having two copies of same library with different names but maybe before fixing CAPI apps problem we should be consistent and keep them both.

Reply:
Got it.